rosbag2_tests/test/rosbag2_tests/test_rosbag2_record_end_to_end.cpp

```diff
@@ -117,7 +118,7 @@ TEST_F(RecordFixture, record_end_to_end_test) {
   wrong_message->string_value = "wrong_content";
 
   auto process_handle = start_execution(
-    "ros2 bag record --output " + root_bag_path_.string() + " /test_topic");
+    "ros2 bag record --max-cache-size 0 --output " + root_bag_path_.string() + " /test_topic");
```

Comment:
Isn't 0 the [default](https://github.com/ros2/rosbag2/blob/d460c1f837a0a3030c26c45c9f563ee966ddb568/ros2bag/ros2bag/verb/record.py#L70)?

Reply:
Oops, I should add this to the comment and the title also. 

0 default = dont use cache and write to disk always, which isnt great for performance. So as per the issue #464 I've also changed the default to 1MB